### PR TITLE
Feature/add integration tests for gec model

### PR DESCRIPTION
**Pull request by ksteimel**
I'll open another PR later today that will convert gec_model to a predictor class.
Let me know if you think more corrections should be included in the tests to verify that switching to an allennlp Predictor class still works.

### CODE REVIEW

File: tests/test_gec_model.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+                                 log=False,
+                                 confidence=0,
+                                 del_confidence=0,
+                                 is_ensemble=0,
+                                 weigths=None)
+        final_batch, total_updates = gec_model.handle_batch(self.input_data)
+        # subject verb agreement is not fixed in the second sentence when predicting using GecModel
+        # (i.e.) brown foxes jump
+        assert final_batch == [['I', 'run', 'to','the','stores', 'every', 'day.'],
+                               ['The', 'quick', 'brown','foxes','jumps','over',"Elmo's", 'laziest', 'friend']]
```
[the remark this answers is not in the document]
Comment (ksteimel) [reply tests/test_gec_model.py:62]: it doesn't fix the verb agreement error. the correction count is tallied during prediction. 
I just wanted to make clear that it's expected for there to still be an error in the output.

File: tests/test_gec_model.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    """Test class for GecModel"""
+    def setUp(self):
+        super().setUp()
+        self.vocab_path = "data/output_vocabulary"
+        self.vocab = Vocabulary.from_files(self.vocab_path)
+        self.model_name = "roberta-base"
+        model_url = "https://grammarly-nlp-data-public.s3.amazonaws.com/gector/roberta_1_gectorv2.th"
+        test_fixtures_dir_path = Path(__file__).parent.parent / "test_fixtures"
+        model_path = test_fixtures_dir_path / "roberta_1_gectorv2.th"
+        if not model_path.exists():
```
Comment (ksteimel) [inline tests/test_gec_model.py:29]: This is a bit hacky. I'm downloading the roberta model weights if they don't exist in test_fixtures. 
Once we use an allennlp Predictor, the model archive can be uploded to a public s3 bucket and then the test can just point at the s3 uri.
Until then, this is a way to get predictions from an existing model. If anyone has a better way to do this in mind, I'm totally open to it.

Comment (desilinguist) [reply tests/test_gec_model.py:29]: Seems reasonable to me as a temporary measure.

**Review by desilinguist — APPROVED** [review]
LGTM!

**Review by Frost45 — APPROVED** [review]
Looks great! 🎉

**Pull request merged**